packages/fleather/lib/src/rendering/editor.dart

```diff
@@ -991,8 +990,7 @@ class RenderEditor extends RenderEditableContainerBox
   }
 }
 
-class FleatherVerticalCaretMovementRun
-    extends BidirectionalIterator<TextPosition> {
```

Comment:
Why did you change `BidirectionalIterator` to `Iterator`?

Reply:
It is deprecated